### ISSUE TITLE
[Bug] audiobook not listed in newest after update

### ISSUE THREAD

**Issue by Ed1ks** [MediaBrowser/Emby#3817]
If I put a .mp3 in the audiodirectory it will be listed as newest audiobook. After adding an .lrc file, the audiofile will not be listed as newest anymore.

reproduction:
When a new audio file with existing album metadata is added, Emby uses the album's original added date, not the new file's date, so the single audiobook doesn't appear in "latest audiobooks."

Subtitle files don’t directly affect this, but they trigger a metadata reload.

**Comment by LukePulverenti** [comment]
HI, I cannot reproduce this. Are you still having an issue with this? Have you checked the date added value of the audiobook in the metadata editor?

**Comment by Ed1ks** [comment]
Yes, I have checked the date added value. Its the same before and after it vanishes.
![image](https://github.com/MediaBrowser/Emby/assets/46250296/2ef7b6d7-6102-4b21-bdf7-0760fe19978c)

See, its not in newest category
![image](https://github.com/MediaBrowser/Emby/assets/46250296/88ad3ad1-b16b-42e5-9073-70ebb8b3886c)


but BEFORE putting an .lrc in the same directory, the audio IS shown in this category.

**Comment by LukePulverenti** [comment]
Can you please show the complete screenshots rather than cutouts? Thanks.

**Comment by Ed1ks** [comment]
Sure:
![image](https://github.com/MediaBrowser/Emby/assets/46250296/d80faed7-0f80-41a0-865f-9472c5bdad48)

![image](https://github.com/MediaBrowser/Emby/assets/46250296/4b7c6f78-600e-4a08-a9a0-8cbec26a554b)

**Comment by Ed1ks** [comment]
Sure:

![image](https://github.com/MediaBrowser/Emby/assets/46250296/9545ce82-9e7c-45c2-a065-fba137e601d0)


![image](https://github.com/MediaBrowser/Emby/assets/46250296/78f4d1c0-237e-4921-b510-a28dbf0dfa8c)

**Comment by LukePulverenti** [comment]
Did you check the date added value of all the other ones showing up in latest? Maybe they have newer dates.

**Comment by Ed1ks** [comment]
Yes I have checked, and I can confirm that the newest doesnt show up.

The "Date Added" sorted table in Media library -> Folders, shows me the vanished on top - with the most recent date.

**Comment by Ed1ks** [comment]
@LukePulverenti 
After further investigation I could find how to reproduce the bug.
It seems that not the .lrc file makes emby not recognizing, but the Artist / Album Artist.
My flow is the following:
1. add mp3 file
2. audio file gets recognized and show in emby
3. I add metadata inside of the file (not in emby, but into the file directly)
4. I add the .lrc file
5. Emby recognize the .lrc file and updates the file (reload)
6. As now Artist and Album Artist are included in file, the new audio file is not shown.

I tested by adding the audiobook without any .lrc and added metadata one by one before adding it to media directory.
After adding Artist and Album Artist and uploading the file, emby show the file in "Folders", but not in the latest Audio Books category:

![image](https://github.com/MediaBrowser/Emby/assets/46250296/6d0a3658-4ff5-40bc-9ae6-f08e86f4cb24)

![image](https://github.com/MediaBrowser/Emby/assets/46250296/a98748b5-d702-418b-9ce9-69f3c01ad7ec)


I would appreciate a bug fix

**Comment by Ed1ks** [comment]
When a new audio file with existing album metadata is added, Emby uses the album's original added date, not the new file's date, so the single audiobook doesn't appear in "latest audiobooks."

**Comment by LukePulverenti** [comment]
Hi, have you updated to Emby Server 4.8.10? Has that helped?

**Comment by Ed1ks** [comment]
> Hi, have you updated to Emby Server 4.8.10? Has that helped?

Hi,

yes I have 4.8.10 but issue still exists. Nothing has changed.
Have you tried to reproduce this bug?

**Comment by LukePulverenti** [comment]
Hi, have you updated to Emby Server 4.9? has that helped?

**Comment by LukePulverenti** [comment]
@Ed1ks ?
